docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -80,5 +91,51 @@ Fleet UI:
 
 The Apple Business Manager server token expires after a year or whenever the account that downloaded the token has their password changed. To renew the token, follow the [instructions documented in this FAQ](https://fleetdm.com/docs/using-fleet/faq#how-can-i-renew-my-apple-business-manager-server-token).
 
+## Disk encryption key
+
+Currently, the disk encryption key refers to the FileVault recovery key only for macOS 10.15+ devices. FileVault allows you to access and recover data on a device without the login credentials. This key is stored by Fleet and can be accessed by Fleet admin, maintainers, and observers to log into a device without its password. An event is tracked in the activity feed when a user views the key using Fleet.
+
+### Viewing a disk encryption key
+
+To view the disk encryption key, select the host on Fleet's manage host page to view host details. In the host's action menu, select Show disk encryption key to view the key.
```

Comment:
```suggestion
To view the disk encryption key, select a host on the **Hosts** page. On the **Host details** page, select **Actions > Show disk encryption key**.
```

I updated the page and button name formatting (bold font) to match what we're using elsewhere in the docs.